### PR TITLE
config: clean up previous community stuff that invalid now

### LINKED ISSUE #134: Clean up previous community stuff that invalid now

### PR DESCRIPTION
**Pull request by tisonkun**
This pull request is based on #133 .

closed #134

### CODE REVIEW

**Comment by tisonkun** [comment]
@Mini256 Please check whether the automatic logic break.

**Comment by Mini256** [comment]
> @Mini256 Please check whether the automatic logic break.

OK, ti-community-bot has been uninstalled from the repository, and TIKV related SIG data will also be hidden from the contributor.tidb.io page.

**Comment by tisonkun** [comment]
@Mini256 IIRC if a PR in pingcap/tidb doesn't have sig label, formerly people in TiKV SIGs can review it. Of course now they cannot, but does it cause panic?

**Comment by Mini256** [comment]
> @Mini256 IIRC if a PR in pingcap/tidb doesn't have sig label, formerly people in TiKV SIGs can review it. Of course, now they cannot, but does it cause panic?

TiKV SIGs no longer be able to review the PR without sig label in pingcap/tidb, but it will not cause panic.

**Comment by tisonkun** [comment]
> > @Mini256 IIRC if a PR in pingcap/tidb doesn't have sig label, formerly people in TiKV SIGs can review it. Of course, now they cannot, but does it cause panic?
> 
> TiKV SIGs no longer be able to review the PR without sig label in pingcap/tidb, but it will not cause panic.

thanks!

/assign @winkyao 

would you mind to help on merging this PR?

**Comment by tisonkun** [comment]
/assign @BusyJay 

would you mind to help on reviewing and merging this PR?

File: README.md [hunk 1]
```diff
@@ -6,29 +6,22 @@ Your journey of becoming a contributor and committer starts from here: improving
...
-
-+ WeChat: [https://github.com/tikv/tikv/blob/master/README.md#WeChat](https://github.com/tikv/tikv/blob/master/README.md#WeChat)
-
-+ Twitter: [https://twitter.com/tikvproject](https://twitter.com/tikvproject)
-
-+ Blog: [https://tikv.org/blog/](https://tikv.org/blog/)
-
-+ Reddit: [https://www.reddit.com/r/TiKV](https://www.reddit.com/r/TiKV)
+* Slack: [https://tikv-wg.slack.com](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE)
+* Discourse: [Discourse](https://internals.tidb.io/c/storage)
```
Comment (skyzh) [inline README.md:20]: I think it's better to use "TiDB internals" here. Maybe not everyone knows what Discourse is...

Comment (tisonkun) [reply README.md:20]: Make sense that "Discourse" may not deliver a good image. But it is a bit tricky using "TiDB Internals", I'd prefer

```suggestion
* Forum: [developer discussion forum](https://internals.tidb.io/c/storage)
```

as in https://github.com/rust-lang/rfcs

Comment (skyzh) [reply README.md:20]: Looks good.

Comment (tisonkun) [reply README.md]: @skyzh thanks for your review! May you send a review reaction or help on merging?

**Review by skyzh — APPROVED** [review]

**Pull request merged**